schemas/bursar-export/legacy/legacyBursarFeeFinesTypeMapping.json

```diff
@@ -11,7 +11,7 @@
       "itemType": {
         "type": "string",
         "minLength": 1,
-        "maxLength": 12
+        "maxLength": 1
```

Comment:
Is this correct? Item type should be the only one symbol?

Reply:
I think that is a mistake. I will correct this.